census/tests/test_census.py

```diff
@@ -121,6 +121,67 @@ def test_la_canada_2015(self):
         )
 
 
+class TestCodedValues(CensusTestCase):
+    """
+    Tests for handling coded values, like -666666666 and -999999999.
+    """
+    def test_handle_666666666(self):
+        """
+        Test the default behavior of handling -666666666 values, which is to
+        cast them to null.
+        """
+        # This call should return a value of -666666666
+        return_val = self._client.acs5.state_county_tract('B19081_001E',
```

Comment:
Do we want this to be the default behavior? I think the default should be the behavior of cast_null=True, but I'm persuadable.